lib/srv/authhandlers.go

```diff
@@ -109,15 +109,17 @@ func (h *AuthHandlers) CheckPortForward(addr string, ctx *ServerContext) error {
 		userErrorMessage := "port forwarding not allowed"
 
 		// emit port forward failure event
-		h.AuditLog.EmitAuditEvent(events.PortForwardFailure, events.EventFields{
+		if err := h.AuditLog.EmitAuditEvent(events.PortForwardFailure, events.EventFields{
 			events.PortForwardAddr:    addr,
 			events.PortForwardSuccess: false,
 			events.PortForwardErr:     systemErrorMessage,
 			events.EventLogin:         ctx.Identity.Login,
 			events.EventUser:          ctx.Identity.TeleportUser,
 			events.LocalAddr:          ctx.Conn.LocalAddr().String(),
 			events.RemoteAddr:         ctx.Conn.RemoteAddr().String(),
-		})
+		}); err != nil {
+			h.Warnf("Failed to emit port forward deny audit event: %v", err)
+		}
```

Comment:
nit: This seems quite long for the one-line `if err := blah(); err != nil` type syntax. I don't know enough about Go reviews to say whether it's something that should generally be avoided or not though.

Reply:
Logically it's one long line, yeah.
But practically it's easy enough to distinguish the different parts here. I've seen many such snippets, it's generally acceptable.